### PR TITLE
Updating script to escape env vars in .bashrc

### PR DESCRIPTION
**Pull request by tsanders-rh**
@BenHall @mhausenblas @madorn This will fix the GOBIN issue in the 3.11 env.

### CODE REVIEW

**Comment by BenHall** [comment]
Thanks :)

**Comment by tsanders-rh** [comment]
@BenHall - I added the set-env update to this PR. Assume merge and get the ansible-op course updated in production.  I realize that the image will not be rebuilt in time, but we will be ready for the next respin.  Thanks!

File: ansibleop/ansible-operator-overview/set-env.sh [hunk 1]
```diff
@@ -1,2 +1,5 @@
+export GOPATH=/root/tutorial/go
+export GOBIN=/root/tutorial/go/bin
+
```
Comment (madorn) [inline ansibleop/ansible-operator-overview/set-env.sh]:
```suggestion
wget https://github.com/operator-framework/operator-sdk/releases/download/v0.3.0/operator-sdk-v0.3.0-x86_64-linux-gnu -O /root/tutorial/go/bin/operator-sdk
chmod +x /root/tutorial/go/bin/operator-sdk
```

**Comment by tsanders-rh** [comment]
Since this isn't going to get merged in time for the workshop & it sounds like workaround have been set; I am going to close this PR and reopen with the needed changes to the operator-sdk installation script for the 3.11 env.

**Pull request closed without merging**